actors/builtin/market/market_test.go

```diff
@@ -1619,13 +1706,19 @@ func TestCronTickDealSlashing(t *testing.T) {
 		rt.SetEpoch(current)
 		actor.cronTickNoChangeBalances(rt, client, provider)
 
+		// a second cron tick for the same epoch should not change anything
+		actor.cronTickNoChangeBalances(rt, client, provider)
```

Comment:
There are no checks here to ensure that a second cron tick is idempotent

Reply:
@wadeAlexC I've added a test to check that the `DealsByEpoch` cid does not change.